buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -286,7 +286,11 @@ def build_with_slack(DOWNSTREAM_JOB_NAME, ghprbGhRepository, ghprbActualCommit,
     // Set Github Commit Status
     if (ghprbActualCommit) {
         node(SETUP_LABEL) {
-            set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, BUILD_URL, 'PENDING', "Build Started")
+            try {
+                retry_and_delay({set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, BUILD_URL, 'PENDING', "Build Started")})
+            } catch (e) {
+                println "set_build_status failed"
```

Comment:
Should there be distinct messages for each of these cases, to make it easy to tell which one failed?

Reply:
Sure. Thanks for checking! :)

Reply:
Hi @pshipton! I just amended the messages. Please let me know if there is any other change. Thanks for checking it again!